crates/lib3h/src/gateway/gateway_transport.rs

```diff
@@ -228,7 +239,7 @@ impl P2pGateway {
             };
 
         trace!(
-            "({}).priv_low_level_send message from '{}' to '{}'",
+            "({}).priv_low_level_send message from {} to {}",
```

Comment:
???

Reply:
Dunno what happened there.  I don't feel like I did that, came from some weird merge.  I'll put them back if you like